Features/Standard/platformui.feature

```diff
@@ -0,0 +1,132 @@
+Feature: Login on PlatformUI
```

Comment:
Is it really what is tested here ? Besides the first scenario, they look more like navigation scenarios.